indico/modules/rb_new/client/js/common/timeline/ElasticTimeline.jsx

```diff
@@ -89,7 +89,7 @@ export default class ElasticTimeline extends React.Component {
                 preConflicts: preConflicts[dt] || [],
                 blockings: blockings[dt] || [],
                 nonbookablePeriods: nonbookablePeriods[dt] || [],
-                unbookableHours: unbookableHours || [],
+                unbookableHours: (dateRange.length !== 0 && !_.includes(dateRange, dt)) ? [] : (unbookableHours || []),
```

Comment:
You can use `Array.prototype.includes` instead of using `_.includes`

Reply:
`dateRange.includes(dt)` to be exact ;)

Reply:
Yeah, yeah. That is what I meant :P